tests/harness/executor/cases.go

```diff
@@ -1175,15 +1175,15 @@ var mapCases = []TestCase{
 
 var oneofCases = []TestCase{
 	{"oneof - none - valid", &cases.OneOfNone{O: &cases.OneOfNone_X{X: "foo"}}, 0},
-	{"oneof - none - valid (empty)", &cases.OneOfNone{}, 0},
+	{"oneof - none - invalid (empty)", &cases.OneOfNone{}, 1},
 
 	{"oneof - field - valid (X)", &cases.OneOf{O: &cases.OneOf_X{X: "foobar"}}, 0},
 	{"oneof - field - valid (Y)", &cases.OneOf{O: &cases.OneOf_Y{Y: 123}}, 0},
 	{"oneof - field - valid (Z)", &cases.OneOf{O: &cases.OneOf_Z{Z: &cases.TestOneOfMsg{Val: true}}}, 0},
-	{"oneof - field - valid (empty)", &cases.OneOf{}, 0},
+	{"oneof - field - invalid (empty)", &cases.OneOf{}, 1},
```

Comment:
I think we want this to stay valid. Oneofs can be specified as required but by default aren't: https://github.com/envoyproxy/protoc-gen-validate/blob/4f41f10dde19ec4dd9dd8cf6f7537139f52fb58b/tests/harness/cases/oneofs.proto#L28